docs/source/quick_start.rst

```diff
@@ -0,0 +1,199 @@
+============
```

Comment:
Please use the heading scheme in the comments at the top of https://raw.githubusercontent.com/kokkos/kokkos-core-wiki/main/docs/source/templates/class_api.rst

(I realized in hindsight it's not the most accessible as it is in a comment and not rendered)

Reply:
Done!